lib/matplotlib/transforms.py

```diff
@@ -932,34 +927,32 @@ def update_from_data_y(self, y, ignore=None):
         ----------
         y : `~numpy.ndarray`
             Array of y-values.
-
         ignore : bool, optional
-           - When ``True``, ignore the existing bounds of the `Bbox`.
-           - When ``False``, include the existing bounds of the `Bbox`.
-           - When ``None``, use the last value passed to :meth:`ignore`.
+            - When ``True``, ignore the existing bounds of the `Bbox`.
+            - When ``False``, include the existing bounds of the `Bbox`.
+            - When ``None``, use the last value passed to :meth:`ignore`.
         """
         y = np.ravel(y)
         self.update_from_data_xy(np.column_stack([np.ones(y.size), y]),
                                  ignore=ignore, updatex=False)
 
     def update_from_data_xy(self, xy, ignore=None, updatex=True, updatey=True):
         """
-        Update the bounds of the `Bbox` based on the passed in data. After
-        updating, the bounds will have positive *width* and *height*;
+        Update the `Bbox` bounds based on the passed in *xy* coordinates.
+
+        After updating, the bounds will have positive *width* and *height*;
         *x0* and *y0* will be the minimal values.
 
         Parameters
         ----------
-        xy : `~numpy.ndarray`
-            A numpy array of 2D points.
-
+        xy : (N, 2) array-like
+            Array of (x, y) coordinates.
```

Comment:
Kind of inconsistent/imprecise: array vs. array-like
```suggestion
            The (x, y) coordinates.
```